docs/csharp/event-pattern.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: The Standard .NET Event Pattern | Microsoft Docs
```

Comment:
This is out of scope for metadata changes, but this is a confusing title. I'd change it to "Standard .NET event patterns". In its current form, it seems ambiguous -- either it may be related to .NET Standard, or there's only one event pattern. Neither is accurate.